### DISCUSSION TITLE
Font not supported? Cannot recognize these digits.

### DISCUSSION THREAD

**Discussion by ebivan** [jomjol/AI-on-the-edge-device#1521] [Q&A]
Hello there,
I wonder if I got a very exotic power meter here, the font seems to complicate things:

![grafik](https://user-images.githubusercontent.com/29729742/206399152-ea02c104-e88f-465d-96b2-9990cacde332.png)

![grafik](https://user-images.githubusercontent.com/29729742/206400718-27ba9053-c6a4-47af-b1c0-e8c9b6edea90.png)

Of course I tried the ROIs with locked aspect ration and synchronized x,y etc too, but this seems to give best results so far. 
I also tried all all of the available recognition modells too, but no luck with any of them:
![grafik](https://user-images.githubusercontent.com/29729742/206399665-f0fc8506-477b-4e47-a67a-04ba0aa3d01a.png)

I got a pretty regular gas meter, had it up and running in 15 min, but for this power meter, I have been trying for hours and no luck so far. What can I do?

**Comment by caco3** [comment]
The contrast also is not very well (reflections of the LED).

Did you check if this digits type is already in the model? See here https://haverland.github.io/Tenth-of-step-of-a-meter-digit/

If your type is missing, you can collect them and provide them for training the model, see https://github.com/jomjol/AI-on-the-edge-device/wiki/Learn-models-with-your-own-images

**Comment by jomjol** [comment]
Problem is the image quality - even for human eye hard to read:
![grafik](https://user-images.githubusercontent.com/30766535/206757310-0924ba71-21c6-4aae-9f8c-ba934e6dcb96.png)

**Comment by ebivan** [comment]
Thanks for your replies! I will try to adjust the Camara setup to get the image quality up and if that alone doesnt do the trick, I will try to capture the digits.
